views/_layout.njk

```diff
@@ -87,31 +77,22 @@
     <div class="container">
       {% block alerts %}      
         <div class="container">
-          <div class="row">
-            <div class="col-lg-6 mx-auto">
-              <div id="alertError" class="alert alert-danger collapse">
-                <a href="#" id="closeError" class="close">&times;</a>
-                <strong>Error:</strong> {{message}}
-              </div>
-            </div>
-          </div>
-          <div class="row">
-            <div class="col-lg-6 mx-auto">
-              <div id="alertSuccess" class="alert alert-success collapse">
-                <a href="#" id="closeSuccess" class="close">&times;</a>
-                <strong>Success:</strong> {{message}}
-              </div>
-            </div>
-          </div>
-        </div> <!-- /alerts container -->
         {% if messages.error %} 
           {% for msg in messages.error%}
           <div class="alert alert-danger alert-dismissible">
             <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
             {{msg}}
           </div>
           {% endfor %}
+        {% elseif messages.success %}
```

Comment:
You beat me to the success messages haha! I implemented the same thing as you have but a little bit differently. The only problem I think I see here is if I want to display both success messages and error messaged. With a `if () else if ()` setup it will allow for only one type to be displayed. To solve this I have implemented it this way on my branch: 

```
      {% if messages.success %} 
          {% for msg in messages.success%}
          <div class="alert alert-success alert-dismissible">
            <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
            {{msg}}
          </div>
          {% endfor %}
        {% endif %}
        {% if messages.error %} 
          {% for msg in messages.error%}
          <div class="alert alert-danger alert-dismissible">
            <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
            {{msg}}
          </div>
          {% endfor %}
        {% endif %}
```

Reply:
I didn't think of that! Good call!